File: misterpasta.client/models/Image.ts
export class Image {
  public ImageId: number;
  public ProductId: number;
  public ImageUrl: string;
  

  constructor(ImageId: number, ProductId: number, ImageUrl: string) {
    this.ImageId = ImageId;
    this.ProductId = ProductId;
    this.ImageUrl = ImageUrl;
  }
}

File: MisterPasta.Server/Models/Image.cs
namespace MisterPasta.Server.Models
{
    public class Image
    {
        public int ImageId { get; set; }
        public int ProductId { get; set; } // Foreign Key naar Product
        public required string ImageUrl { get; set; }
        public Product? Product { get; set; }
    }
}
